src/main/java/org/frankframework/frankdoc/model/FrankDocModel.java

```diff
@@ -189,7 +189,7 @@ void checkRootElementsMatchDigesterRules(Set<String> rootRoleNames) {
 				.collect(Collectors.toList());
 		for(RootFrankElement root: roots) {
 			if(! rootRoleNames.contains(root.getRoleName())) {
-				log.warn("Root FrankElement [{}] does not have a matching pattern in digester-rules.xml", root.toString());
+				log.error("Root FrankElement [{}] does not have a matching pattern in digester-rules.xml", root.toString());
```

Comment:
Dit lijkt me geen reden om de build te laten falen

Reply:
Als deze fout optreedt, dan zijn er corner cases waarbij het matchen van digester-rules.xml patterns niet goed werkt. Om het patroon "configuration/adapter" te matchen heb je class "Configuration" nodig als RootFrankElement en in `digester-rules.xml` de volgende rule:

```
<rule pattern="configuration" factory="null"/>
```

Niels en Gerrit, blijven jullie met deze informatie bij jullie standpunt?

Reply:
Hij is weer een warning.

Reply:
Ik heb hem weer error gemaakt. Niels en ik hadden een misverstand over de betekenis van deze fout. Ik heb de tekst van de foutmelding aangepast. Het punt is dat digester-rules.xml een rule moet hebben met pattern "configuration", omdat anders het verwerken van digester-rules.xml onbetrouwbaar is. Deze situatie is ongewenst. Ik heb daar een kaartje voor aangemaakt op het Kanban bord.

Reply:
Ik heb nog eens goed naar de code voor digester-rules.xml pattern matching gekeken. De code werkt nog steeds als de rule van pattern "configuration" wordt weggehaald. Het is wel zo dat class "Configuration" matcht met pattern component "configuration", omdat het de root class is. Dit wordt geïmplementeerd door RootClassName.getRoleName(). Deze functie transformeert de simple class name naar lower-case.

Het is dus zo dat class Configuration met pattern component "configuration" correspondeert. Om deze reden leek het me logisch om een pattern "configuration" in digester-rules.xml moet zijn. Dit is eigenlijk een validatie dat de root class inderdaad "Configuration" is. Maar de rule "configuration" kan dus weg zonder dat de werking van de Frank!Doc verandert.

Wat zal ik met de code doen? Log level verlagen of de hele test weghalen?

Reply:
Haal de test maar weg

Reply:
De test is weg.